specification/apimanagement/resource-manager/readme.typescript.md

```diff
@@ -9,6 +9,8 @@ typescript:
   package-name: "@azure/arm-apimanagement"
   output-folder: "$(typescript-sdks-folder)/sdk/apimanagement/arm-apimanagement"
   generate-metadata: true
+  modelerfour: 
+    lenient-model-deduplication: true
```

Comment:
I don't think this is a good solution to fix duplicate, could you unify the reference versions from common-type?

Reply:
Same for readme.go.md, please

Reply:
This is rolled back